apps/frontend/src/pages/profile/profile.tsx

```diff
@@ -28,6 +28,7 @@ const Profile: React.FC = () => {
 		defaultValues: {
 			firstName: user.firstName,
 			lastName: user.lastName,
+			nickname: user.nickname,
```

Comment:
```suggestion
			nickname: user.nickname ?? "",
```
It cause error in react-hook-form